src/Integration/Binding/CSharpVBBindingOperation.cs

```diff
@@ -119,6 +125,21 @@ public void Commit()
             }
         }
 
+        private void AddAdditionalFile()
+        {
+            try
+            {
+                var projectSystem = serviceProvider.GetService<IProjectSystemHelper>();
+                projectSystem.AssertLocalServiceIsNotNull();
+                projectSystem.RemoveFileFromProject(initializedProject, cSharpVBBindingConfig.AdditionalFile.Path);
```

Comment:
This needs some explanation.

Reply:
There are two uses cases here:
1. It's possible that the additional file is set property, but the ruleset is out of date, which will trigger the binding. So to avoid receiving a "file already exists" -- I remove the file first.
2. It's possible that the additional file exists but has a wrong type, e.g. Compile/Content, and not AdditionalFiles. In which case removing and re-adding it solves the problem.

I will add unit tests to cover those 2 cases.

Reply:
The explanation needs to be in the code.